BangazonAPI/Models/EmployeeTraining.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class EmployeeTraining
+    {
+
+        public int Id { get; set; }
+        public int EmployeeId { get; set; }
+        public int TrainingProgram { get; set; }
```

Comment:
Do we ever need to expose an `EmployeeTraining` instance to the user? I can't think of a situation where we would, but feel free to correct me.

If we want to represent the many-to-many relationship between `Employees` and `TrainingProgram` we should have a list on Employee of type `TrainingProgram` and a list on `TrainingProgram` of type `Employee`

Reply:
added

Reply:
So actually I don't think we need this model at all, based on the tickets. We could sub it out for just having a list of Employees on Training Programs and a list of Training Programs on Employees. Sorry if that was unclear!

Reply:
Sorry, I read that as "I can think...." instead of "I can't think"...   Now deleted.